### ISSUE TITLE
Custom model data not supported by randomize icon property

### ISSUE THREAD

**Issue by grisstyl** [elBukkit/MagicPlugin#635]
I can't get custom model data to work with the `randomize_icon` property of wands. 

Ex:
```
randomizedpotterwand:
  icon: wooden_hoe:{CustomModelData:1}
  randomize_icon: wooden_hoe:{CustomModelData:3}
```

Comes out as a basic wooden hoe that turns into the correct custom model data attached to the icon property.

**Comment by NathanWolf** [comment]
I *think* this may have been fixed by the fix for #639 - can you please let me know if not?

**Comment by grisstyl** [comment]
Yep! Everything looks to be working correctly. Thanks Nathan!
